File: Advanced/BottomNavigationViewDemo/app/src/main/java/edu/cs4730/bottomnavigationviewdemo/OneFragment.java
package edu.cs4730.bottomnavigationviewdemo;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import edu.cs4730.bottomnavigationviewdemo.databinding.FragmentOneBinding;

/**
 * A simple {@link Fragment} subclass.
 */
public class OneFragment extends Fragment {

    FragmentOneBinding binding;

    public OneFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        binding = FragmentOneBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

}

File: Advanced/BottomNavigationViewDemo_kt/app/src/main/java/edu/cs4730/bottomnavigationviewdemo_kt/OneFragment.kt
package edu.cs4730.bottomnavigationviewdemo_kt

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import edu.cs4730.bottomnavigationviewdemo_kt.databinding.FragmentOneBinding

/**
 * A simple [Fragment] subclass.
 */
class OneFragment : Fragment() {
    lateinit var binding: FragmentOneBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
    ): View {
        // Inflate the layout for this fragment
        binding = FragmentOneBinding.inflate(inflater, container, false)
        return binding.root
    }
}